projects/plugins/crm/changelog/fix-crm-3440-add_links_to_dash_sales_funnel

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+Dashboard: Sales Funnel now links to contact list view.
```

Comment:
I think this got mixed up with another branch.

Reply:
Indeed; not sure what happened. Repushed with just the relevant commits.